chart/kubeapps/values.yaml

```diff
@@ -19,6 +19,27 @@ allowNamespaceDiscovery: true
 ## Enable IPv6 Configuration for Nginx
 enableIPv6: false
 
+## clusters can be configured with a list of clusters that Kubeapps can target for deployments.
+## This is an alpha feature and the semantics are expected to change slightly in future releases.
+## If empty, Kubeapps will target the cluster on which it is installed only.
+## When populated, Kubeapps will currently present the listed clusters in addition to the default
+## cluster on which Kubeapps is installed. In the future a populated list here will be definitive, so that
+## it is possible to configure Kubeapps to target other clusters only (ie. not target the cluster on which
+## Kubeapps is installed), as per https://github.com/kubeapps/kubeapps/issues/1942 .
+## The base64-encoded certificateAuthorityData can be obtained from the additional cluster's kube config
+## file, for example:
+## kubectl --kubeconfig ~/.kube/kind-config-kubeapps-additional config view --raw -o jsonpath='{.clusters[0].cluster.certificate-authority-data}'
+#
+# clusters:
+# - name: second-cluster
+#   apiServiceURL: https://second-cluster:6443
```

Comment:
after renaming this to `clusters`, it's a bit confusing the sample data (which implies additional clusters)

Reply:
Yeah, that's what I tried to make explicit in the preceding comment? I'm renaming it now so that we don't release and then change the value name for #1942.